packages/@aws-cdk/cx-api/README.md

```diff
@@ -3,16 +3,14 @@
 
 ---
 
-![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
-
-> **This is a _developer preview_ (public beta) module.**
->
-> All classes with the `Cfn` prefix in this module ([CFN Resources](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib))
-> are auto-generated from CloudFormation. They are stable and safe to use.
->
-> However, all other classes, i.e., higher level constructs, are under active development and subject to non-backward
-> compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
-> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+![cfn-resources: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
```

Comment:
does this make sense for this module? (there's a few others... I just pushed one to the `cloud-assembly-schema`.... doesn't feel right.

It's even confusing for `custom-resources` and any other documented module without a documented `cfn*` class. what do you think?

Reply:
Yeah, I had considered that as well, but didn't really want to make the logic even more complicated. But I can. Let me see.